.github/ISSUE_TEMPLATE/bug_template.md

```diff
@@ -21,7 +21,7 @@ A clear and concise description of what happening.
 * OS version: [e.g. 18.04]
 * Architecture: [e.g. arm64]
 * nDPI version or commit hash: [e.g. 4.0-stable or 937357e4bc55610f116f66d15a8e0fc1e260c02c].
-* nDPI compilation flags used: if you are building from source [e.g. --with-pcre --disable-gcrypt].
+* nDPI compilation flags used: if you are building from source [e.g. --with-pcre --with-host-libgcrypt].
```

Comment:
Just one nit: using "host" word in configure parameters make me think of cross-compilation, which is not the case here. Not strong opinion on that but what about using a different name?

Reply:
> Just one nit: using "host" word in configure parameters make me think of cross-compilation, which is not the case here. Not strong opinion on that but what about using a different name?

What about `local` instead of `host` ?